core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@magenta/core",
-  "version": "0.0.9",
+  "version": "0.0.13",
```

Comment:
We've kind of being doing versioning and publishing ad hoc, but I propose we move toward:
- Test changes locally using `yarn link` so you don't have to publish a package to use it when developing dependent packages.
- Do PRs without bumping version numbers.
- Once things have been reviewed and checked in, do `yarn publish`, which will generate a commit that just bumps the version number and publishes to npm.
- Define a `prepublish` task for all our packages that runs lint, test, etc.

This aligns us with how we do Python releases and generally makes sure code is reviewed before it's published.

Thoughts?

Reply:
That sounds good. I wasn't sure how to test that the core changes work in music_rnn, etc. How does yarn link do that?

Reply:
https://yarnpkg.com/lang/en/docs/cli/link/

In core, you do `yarn link`, which creates a global symlink to make it available to other packages. In music_rnn, you do `yarn link @magenta/core`, which instructs it to use the symlink version, not the installed version. You have to redo this after every time you do `yarn install`, so it's a little annoying, but I think it's better than publishing lots of extra versions.